pkgs/by-name/hw/hwinfo/package.nix

```diff
@@ -30,9 +30,13 @@ stdenv.mkDerivation (finalAttrs: {
   ];
 
   postPatch = ''
-    substituteInPlace Makefile --replace "/sbin" "/bin" --replace "/usr/" "/"
-    substituteInPlace src/isdn/cdb/Makefile --replace "lex isdn_cdb.lex" "flex isdn_cdb.lex"
-    substituteInPlace hwinfo.pc.in --replace "prefix=/usr" "prefix=$out"
+    substituteInPlace Makefile \
+      --replace-fail "/sbin" "/bin" \
+      --replace-fail "/usr/" "/"
+    substituteInPlace src/isdn/cdb/Makefile \
+      --replace-fail "lex isdn_cdb.lex" "flex isdn_cdb.lex"
+    substituteInPlace hwinfo.pc.in \
+      --replace-fail "prefix=/usr" "prefix=$out"
```

Comment:
This is not a pure formatting change, it also changes "--replace" into "--replace-fail".

Reply:
Ah sorry, the commit message is indeed misleading 🤦 I'll reword it.

Reply:
Reworded :+1: Thanks for the heads-up. I think the PR is ready for merge